src/ng2-smart-table/components/cell/cell-editors/input-editor.component.ts

```diff
@@ -1,10 +1,10 @@
 import { Component } from '@angular/core';
 
-import { DefaultCellType } from './default-cell-type';
+import { DefaultEditor } from './default-editor';
 
 @Component({
-  selector: 'cell-type-input',
-  styleUrls: ['edit-input.component.scss'],
+  selector: 'input-editor',
+  styleUrls: ['editor.component.scss'],
```

Comment:
- for `styleUrls` and `templateUrl` use relative path like `styleUrls: ['./editor.component.scss']`